smarts/core/sensors.py

```diff
@@ -111,6 +114,22 @@ class DrivableAreaGridMap(NamedTuple):
     data: np.ndarray
 
 
+@dataclass
+class ViaPointData:
```

Comment:
We don't need the `Data` suffix on both for `ViaPointData` and `ViaData`, (it's a dataclass, it's clearly data) and we should rely on modules to provide the necessary namespacing